src/index.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import { SplitFactory } from '@splitsoftware/splitio-react';
 import './index.css';
+import App from './App';
+import sdkConfig from './sdkConfig';
 
 ReactDOM.render(
-  <App />,
+  <SplitFactory config={sdkConfig}>
```

Comment:
Add some comments here, explaining a little bit what's going on (something short, concise, but enough)

Reply:
Also, we could use the isReady flag as this would be the entry point of the app.

Reply:
The isReady flag is actually used inside page components, to show a part of the UI using this flag, and other part not.

Reply:
Maybe we can add a comment? It's a cool feature so it'll be good to make it clear that delaying rendering of your actual page is an option.